docs/pages/identity-governance/entra-id/getting-started.mdx

```diff
@@ -0,0 +1,223 @@
+foreach ($appRole in $appRoles) {
+    # Define the parameters for the role assignment, including the managed identity's principal ID,
+    # the Microsoft Graph service principal's resource ID, and the specific app role ID.
+    $bodyParam = @{
+        PrincipalId = $managedIdentity.Id  # The ID of the managed identity (service principal)
+        ResourceId  = $graphSPN.Id         # The ID of the Microsoft Graph service principal
+        AppRoleId   = $appRole.Id          # The ID of the app role being assigned
+    }
+
+    # Create a new app role assignment for the managed identity, granting it the specified permissions.
+    New-MgServicePrincipalAppRoleAssignment -ServicePrincipalId $managedIdentity.Id -BodyParameter $bodyParam
+}
+
+```
+
+</details>
+
+Your identity principal `<Var name="Principal ID" />` now has the necessary permissions to list Applications,
+Directories, and Policies.
+
+Now, to begin integration, run the `tctl plugins install entraid` command.
+
+```code
+$ tctl plugins install entraid \
+    --name entra-id-default \
+    --auth-connector-name entra-id \
+    --default-owner=<Var name="Access List Owner"/> \
+    --auth-server <Var name="example.teleport.sh:443" /> \
+    --use-system-credentials 
+```
+
+The `--name` flag specifies the resource name of the Entra ID plugin. 
+The `--auth-connector-name` flag specifies the name of the auth connector this integration will create.
+The `--default-owner` flag specifies default owners for the Access Lists that will be created
+in Teleport based on the groups imported from the Entra ID.
+The `--use-system-credentials` flag specifies the plugin will use the system credential configured
+for the Auth Service.
+
+The command will generate a configuration script in the current directory 
+from where the `tctl` is invoked. 
+
+</TabItem>
+</Tabs>
+
+## Step 2/3: Configure Entra ID
+
+(!docs/pages/includes/identity-governance/azure-shell.mdx!)
+
+## Step 3/3: Finish plugin installation
+
+<Tabs groupId="entra-guided">
+<TabItem default label="Teleport as OIDC provider (Web UI)" name="entra-oidc-ui">
+
+Copy the Entra ID tenant ID and enterprise application client ID from the script output
+and enter it in the Web UI to finish the integration.
+
+![Entra ID Integration](../../../img/igs/entraid/entra-finish.png)
+
+</TabItem>
+
+<TabItem label="Teleport as OIDC provider (tctl)" name="entra-oidc-tctl">
+
+Copy the Entra ID tenant ID and enterprise application client ID from the script output
+and enter it in the `tctl` to finish the integration.
+
+</TabItem>
+
+<TabItem label="System credential (tctl)" name="entra-system-cred-tctl">
+
+Copy the Entra ID tenant ID and enterprise application client ID from the script output
+and enter it in the `tctl` to finish the integration.
+
+</TabItem>
+</Tabs>
+
+The integration is now configured and the Teleport Entra ID service will start 
+importing resources from Entra ID to Teleport.
+
+## FAQ
```

Comment:
Should we move FAQ into its own .mdx file and link to it in the next steps? Because you could have those questions after following either of the guides. Same for "Access Management" section we'll add next, I think it should be a separate page under Entra section.